components/SpotlightModal/SpotlightModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { FunctionComponent } from 'react';
+import Popup from 'reactjs-popup';
+import { SpotlightModalContent } from './SpotlightModalContent';
+import './styles.css';
```

Comment:
We always do css-in-js for any non-theme code ... using actual css files is a something we leave to FF

Reply:
Deleted the css file in favor of css-in-js